src/article.ml

```diff
@@ -84,14 +92,17 @@ let process_command cmd stack =
       | "def", Num(k) :: obj :: stack ->
         dict_add k obj;
         obj :: stack
-      | "defineConst", Term(t) :: Name(n) :: stack -> Thm(Thm.define_const n t) :: Const(n) :: stack
+      | "defineConst", Term(t) :: Name(n) :: stack ->
+		Term.add_cst n (Term.type_of t);
```

Comment:
Why did you add this?

Reply:
The constants are added to a table, the goals being to remember where they are defined and ultimately to be able to share them between the generated dk files.